.changeset/spotty-pandas-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/shopify-api': patch
```

Comment:
I put this as a patch because I technically changed the type, but in practice it is unchanged. Is the changeset necessary?

Reply:
I think we could do without the changeset - the public API of the package hasn't changed. That being said, we have other releases lined up so this will go out either way :)